### PR TITLE
refactor: Remove unused person update code

### PR DESCRIPTION
**Pull request by tiina303**
## Problem

Transactions was the wrong way to go. Since we're not actively working on this code to change it and roll it out, let's nuke this to avoid confusion.

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: plugin-server/src/worker/ingestion/process-event.ts [hunk 1]
```diff
@@ -232,11 +228,7 @@ export class EventsProcessor {
         propertiesOnce: Properties,
         timestamp: DateTime
     ): Promise<void> {
-        if (this.isNewPersonPropertiesUpdateEnabled(teamId)) {
-            await updatePersonProperties(this.db, teamId, distinctId, properties, propertiesOnce, timestamp)
-        } else {
-            await this.updatePersonPropertiesDeprecated(teamId, distinctId, properties, propertiesOnce)
-        }
+        await this.updatePersonPropertiesDeprecated(teamId, distinctId, properties, propertiesOnce)
```
Comment (yakkomajuri) [inline plugin-server/src/worker/ingestion/process-event.ts:231]: Are we going to keep the name as "deprecated" given we're deleting the old code?

**Review by yakkomajuri — APPROVED** [review]
one comment, otherwise makes sense

File: plugin-server/src/worker/ingestion/process-event.ts (hunk 1 above)
Comment (yakkomajuri) [reply plugin-server/src/worker/ingestion/process-event.ts:231]: I'm happy to merge, but would like to change this soon. If someone other than you or me looks at the codebase they will be very confused as to what the non-deprecated method is. 

Let's at least add a comment. I also don't get the point about the tests. Now that they all pass you shouldn't need this anymore.

Comment (tiina303) [reply plugin-server/src/worker/ingestion/process-event.ts:231]: yes, the tests part was while they hadn't all passed, like you guessed.

**Pull request merged**